frontend/graph/schema.resolvers.go

```diff
@@ -81,14 +82,21 @@ func (r *computePlatformResolver) K8sActualSource(ctx context.Context, obj *mode
 
 // K8sActualSources is the resolver for the k8sActualSources field.
 func (r *computePlatformResolver) K8sActualSources(ctx context.Context, obj *model.ComputePlatform) ([]*model.K8sActualSource, error) {
+	// sourceList, err := services.GetSourceCRDs(ctx)
+	// if err != nil {
+	// 	return nil, err
+	// }
+
+	// Initialize an empty list of K8sActualSource
+	var actualSources []*model.K8sActualSource
+
+	// for _, source := range sourceList {
+	// TODO: remove "InstrumentedApplications" once we're ready to move over to "InstrumentationConfigs" combined with "Source CRDs"
```

Comment:
why dont we use the source crd to manage the instruments source?

Reply:
Because `InstrumentedApplications` has no labels I can select with